deploy/kiali/kiali_cr.yaml

```diff
@@ -723,3 +723,43 @@ spec:
 # When empty, Kiali will try to guess this value from HTTP headers.
 #    ---
 #    web_schema: ""
+
+##########
+#  ---
+#  health_config:
+#
+#
+# rate: A list of configurations health that Kiali apply. Each item contains:
+#   namespace: The name of the namespace to apply. Kiali match the value like a regular expression
+#   kind: The type of the resource to apply. Kiali match the value like a regular expression
+#   name: Tha name of the resource to apply. Kiali match the value like a regular expression
+#   tolerance: A list of tolerances for this configuration. Each item contains:
+#     protocol: The protocol to apply the tolerance (grpc | http | ...) Kiali match the value like a regular expression
+#     direction: The direction to apply the tolerance (inbound | outbound | ...).Kiali match the value like a regular expression
+#     code: The status code to apply the tolerance. Kiali match the value like a regular expression
+#     degraded: The value in % to show a degraded status
+#     failure: The value in % to show a failure status,
+#    ---
+#    rate:
+#      - namespace: "default"
+#        tolerance:
+#          - request_protocol: "http"
+#            direction: "inbound"
+#            code: "400"
+#            degraded: 5
+#            failure: 10
+#      - namespace: default
+#        tolerance:
+#          - request_protocol: "grpc"
+#            code: "1"
+#            degraded: 10
+#            failure: 15
+#      - namespace: bookinfo
+#        kind: app
+#        name: reviews
+#        tolerance:
+#          - request_protocol: "http"
+#            code: "400"
+#            degraded: 0
+#            failure: 5
```

Comment:
The values here must document the actual defaults (do not put in arbitrary examples). Please [see these comments at the top of this kiali_cr.yaml file that explain this](https://github.com/kiali/kiali-operator/blob/v1.21/deploy/kiali/kiali_cr.yaml#L16-L19) to the readers. So when someone reads this kiali_cr.yaml file, they will know what the defaults are by simply looking at the examples here in the comments. So in other words, just replace these example settings with the actual defaults you have in the defaults/main.yaml in this PR.